ci_framework/playbooks/07-admin-setup.yml

```diff
@@ -0,0 +1,18 @@
+- name: Run pre_admin_setup hooks
+  vars:
+    hooks: "{{ pre_operator_build | default([]) }}"
+    step: pre_admin_setup
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Create openstack network elements
+      ansible.builtin.import_role:
+        name: os_net_setup
+
+- name: Run post_admin_setup hooks
+  vars:
+    hooks: "{{ post_operator_build | default([]) }}"
```

Comment:
ditto - please mention that new hook keyword in doc

Reply:
added